src/lib/tabs/tabRelics/RelicFilterBar.tsx

```diff
@@ -239,6 +239,27 @@ export default function RelicFilterBar(props: {
     window.store.getState().setScoringModalOpen(true)
   }
 
+  function selectRecommendationCharacterRelics() {
+    if (!currentlySelectedCharacterId) return
+  
+    const characterPreferredSets = window.store.getState().characters.find(
+      (character) => character.id === currentlySelectedCharacterId,
+    )
```

Comment:
maybe `characterPreferredSets = window.store.getState().getCharacterById(character.id)` would be more concise here?

Reply:
I'm always up for better naming.

Reply:
const characterPreferredSets = window.store.getState().charactersById[currentlySelectedCharacterId]